DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -112,6 +113,7 @@ struct Digit {
   void setEventStatus(uint8_t stat) { mEventStatus = stat; };
   void setStatusFlag(EEventStatus bit, bool value) { mEventStatus |= (value << bit); };
   bool getStatusFlag(EEventStatus bit) const { return bool(mEventStatus << bit); }
+  uint8_t getEventStatusWord() const { return mEventStatus; }
   ClassDefNV(Digit, 4);
```

Comment:
Please increase class version

Reply:
Its sum amplitude from A and C sides in TCM, 18 bits long

Reply:
so, the range is not +-2048?

Reply:
no